.cicd/test.sh

```diff
@@ -184,13 +184,44 @@ function check_creates_template() {
   check_string_existence "--set modules.text2vec-voyageai.enabled=true --set modules.text2vec-voyageai.apiKey=voyageaiApiKey --set modules.reranker-voyageai.enabled=true --set modules.reranker-voyageai.apiKey=voyageaiApiKey" "name: VOYAGEAI_APIKEY"
 
   check_setting_has_value "--set replicas=3 --set env.RAFT_BOOTSTRAP_EXPECT=3" "name: RAFT_JOIN" "value: \"weaviate-0,weaviate-1,weaviate-2\""
+  check_setting_has_value "--set replicas=3 --set env.RAFT_BOOTSTRAP_EXPECT=3" "name: RAFT_BOOTSTRAP_EXPECT" "value: \"3\""
   check_setting_has_value "--set replicas=8 --set env.RAFT_BOOTSTRAP_EXPECT=3" "name: RAFT_JOIN" "value: \"weaviate-0,weaviate-1,weaviate-2\""
```

Comment:
Wouldn't it now make more sense to change this test into (let's verify that if we override the BOOTSTRAP_EXPECT parameter it is still possible? And change the check into:
`check_setting_has_value "--set replicas=8 --set env.RAFT_BOOTSTRAP_EXPECT=5" "name: RAFT_JOIN" "value: \"weaviate-0,weaviate-1,weaviate-2,weaviate-3,weaviate-4\""`

Reply:
Ohhh, good point! maybe I will leave this test, but I will add your test cases as they are actually testing the possibility of overriding those values.

Reply:
I have pushed tests which show that we can override the RAFT values